resources/plugin-changenotes.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    2020-12-24
+    <li>
+        Complete refactor of the Amazon Ion plugin 🎉
```

Comment:
Can you please add some notes about the differences between this version and the previous version?

Are there any features in the previous version that aren't in this version, or vice-versa?

Reply:
I can add more details here.

Reply:
@tgregg FYI

1. This release doesn't publish the new version to the JetBrains public repo just yet. I'll make another PR after this one to make the changes to the GitHub Actions flow to be able to publish automatically from GitHub. We need to get the publish key from IntelliJ and figure out how to safely store that on GitHub. Currently, the way you would upgrade your plugin is by downloading the build artifacts from this merge, in fact you can just download the JAR produced by the GitHub Actions flow right now and test the plugin locally. Its quite easy.
2. I'm not sure how to specifically put this into the Changelog but the changes between the old plugin and the new plugin are pretty massive, I've asked a co-worker who had the old plugin on their IntelliJ to run Code -> Reformat Code with the old plugin and uploaded both onto a Github Gist.
> [V1 Plugin Formatting](https://gist.github.com/belloale/b2fe46b84043508e8356ca85ea6f0c9d)
> [V2 Plugin Formatting](https://gist.github.com/belloale/62e9412643881e69c89a4b9d4f2c6291)

Reply:
This is great, thanks! Some major improvements in there.